infrastructure/cleardata/terragrunt.hcl

```diff
@@ -0,0 +1,83 @@
+remote_state {
+  backend = "s3"
+  generate = {
+    path      = "backend.tf"
+    if_exists = "overwrite"
+  }
+  config = {
+    bucket         = "nc-terraform-private"
+    encrypt        = true
+    key            = "cleardata/${path_relative_to_include()}/terraform.tfstate"
+    region         = "us-east-1"
+    profile        = "cleardata"
+    dynamodb_table = "nc-terraform-lock-table"
+  }
+}
+
+# Generate AWS provider. You may need to add a second in your module if you need public DNS in route53.
```

Comment:
> Since this repo has stuff other than just the updated python, should it be public? Or it doesn't matter? Does it give away any internal stuff, naming, etc?

Only thing I would say it gives away would be the terraform under infrastructure/ which is really just AWS account IDs, subnet CIDRs, and some naming conventions, which I wouldn't really say is necessarily sensitive. I'm open to mirror cloning it and deleting this repo if you have concerns about it, but I don't view that level of info as something that absolutely must be kept private.